Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/onboarding/OnboardingLoadingViewModel.kt

```diff
@@ -24,7 +24,8 @@ class OnboardingLoadingViewModel @AssistedInject constructor(
             !cwaSettings.wasInteroperabilityShownAtLeastOnce -> {
                 navigationEvents.postValue(OnboardingFragmentEvents.ShowInteropDeltaOnboarding)
             }
-            cwaSettings.lastChangelogVersion.value < BuildConfigWrap.VERSION_CODE -> {
+            cwaSettings.lastChangelogVersion.value.toString().substring(0, 3).toInt() <
+                (BuildConfigWrap.VERSION_MAJOR.toString() + BuildConfigWrap.VERSION_MINOR.toString()).toInt() -> {
```

Comment:
I think you can simply check it as:
```kotlin
cwaSettings.lastChangelogVersion.value / 100_00 < BuildConfigWrap.VERSION_CODE / 100_00
```

and please write some unit tests to make sure it works as expected ?